src/timecop.cr

```diff
@@ -4,96 +4,107 @@ require "./timecop/core_ext/*"
 module Timecop
   extend self
 
+  class_property? safe_mode = false
+
   @@stack = [] of TimeStackItem
-  @@safe_mode = false
 
-  # Returns whether or not Timecop is currently frozen/travelled/scaled
-  def frozen?
+  # Returns whether or not `Timecop` is currently frozen/travelled/scaled
```

Comment:
Does this actually return true when time is scaled but not frozen? If so, I’d like to rename this method, and provide one which is less confusing.

Reply:
I don't think it's going to be beneficial since it's a stack, thus you might have couple of different mocks in there. That's a pretty safe way to check whether the mocking takes place at all.

Reply:
This is not a blocker for this pull, I just bring it up because so much of the codebase was churned in this pull request.

All I'm saying is, when I ask an explicit question like `Timecop.frozen?` I don't expect it to return true unless timecop is _frozen_. When the time is scaled, it's not frozen, so returning true is misleading, and someone will eventually trip on this bug.

Reply:
There is a clear information about that in the docs. I'd agree thought that the name could surely be more apt.